src/components/HomePermalink/HomePermalink.jsx

```diff
@@ -1,11 +1,28 @@
-import React from "react";
+import React, { useContext } from "react";
 import Link from "COMPONENTS/Link/";
+import { NavContext as Nav } from "COMPONENTS/Navigation/";
+import { s } from "./";
 
 export default function HomePermalink(){
+
+	//CONTEXT
+	//----------------------------
+
+	const {
+		open
+	} = useContext(Nav).state;
+
+	//RENDER
+	//----------------------------
+
+	const isVisible = !open;
+
 	return (
```

Comment:
rather than do the check inside the return, it makes more sense to either return something or not, y'know?  I'd say do

```
if(isVisible) return (<Link ....... />);
else return null;
```

Reply:
might not need the `else`? either `react` or `preact` gets grumpy, if you don't explicitly return something, but I can't remember which 🤦‍♂

Reply:
react definitely is not please when there's no return! :P